src/server/prod.ts

```diff
@@ -2,10 +2,26 @@ import compression from 'compression';
 import history from 'connect-history-api-fallback';
 import express from 'express';
 import http from 'http';
+import minimist from 'minimist';
 import favicon from 'serve-favicon';
+import sio from 'socket.io';
+
+import { WebSocketProxyUDPServer } from './network/network';
+import { WebSocketServer } from './network/web_socket_server';
+
+const args = minimist(process.argv.slice(2));
+const teamAAddress = args.team_a || '239.226.152.162';
```

Comment:
Where do these defaults come from?

Reply:
I think I stole that one from NUClear, the other is just +1. No idea if there is anything special about them except that they are in a good range of IPs [https://en.wikipedia.org/wiki/Multicast_address#IPv4](https://en.wikipedia.org/wiki/Multicast_address#IPv4)